src/toil/lib/ec2.py

```diff
@@ -244,7 +258,12 @@ def spotRequestNotFound(e):
         if instance_ids:
             # This next line is the reason we batch. It's so we can get multiple instances in
             # a single request.
-            yield ec2.get_only_instances(instance_ids)
+            for instance_id in instance_ids:
+                for attempt in retry_ec2():
+                    with attempt:
+                        # Increase hop limit from 1 to use Instance Metadata V2
+                        boto3_ec2.modify_instance_metadata_options(InstanceId=instance_id, HttpPutResponseHopLimit=3)
```

Comment:
This should just be a call to `increase_instance_hop_limit()`, right?

Reply:
I don't think this is a boto3 call, and this seems like the main way to adjust HTTP related settings in boto3